app/components/cookiesyncer-frame.js

```diff
@@ -0,0 +1,11 @@
+import {inject as service} from '@ember/service';
+import {computed} from '@ember/object';
+import config from '../config/environment';
+import Component from '@ember/component';
+
+export default Component.extend({
+	currentUser: service(),
+	tagName: '',
+	servicesDomain: computed(() => config.APP.servicesExternalHost),
```

Comment:
since config is not dynamic here, I don't think you need a computed prop here
servicesDomain: config.APP.servicesExternalHost - should work just fine

Reply:
I think we tried this before and it didn't work.

Reply:
kk